packages/blade/src/components/ChatMessage/_decision.md

```diff
@@ -0,0 +1,85 @@
+# ChatMessage Decisions
+
+- [Design](#design)
+- [ChatMessage Component](#ChatMessage-component)
+  - [ChatMessage API](#ChatMessage-api)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=100413-32686&t=n9A7LztwEkIsly3v-0)
+
+## ChatMessage Component
+
+This will be our main component that will be used to render the chat bubble.
+
+## ChatMessage API
+
+| Prop                   | Type                                        | Default | Required | Description                                                                                                                                                     |
+| ---------------------- | ------------------------------------------- | ------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| messageType          | 'last' or  'default'                                | default   | No       | If the message is the last message in the chat and if this prop is enabled we will add decorations messageBubble                                                |
+| senderType          | 'self' or 'other'                              | self   | No       | we will add  different styles based on this bubble                              |
+| isLoading              | Boolean                                     | false   | No       | If the message is loading, we will add a loading animation to the chat bubble                                                                                   |
+| validationState                | 'error' or 'none'                              | none   | No       |  if validation state is error we will show error decoration and message|
+| errorText              | String                                      | null    | No       | If the message is an error message, we will show the error message in the chat bubble                                                                                   |
+| onClick           | Function                                    | null    | No       | callback to be called when ever component is clicked.                                                                                                                          |
```

Comment:
Does it need to be interactive/clickable? If we render a Card inside the slot, it would support its own `onClick` prop.

Reply:
Yes, in case of an error, we need to support the onClick prop since children accepts JSX. It should support its own onClick, but in that case, we need to stop event bubbling.